### PR TITLE
exclude-this and exclude

### PR DESCRIPTION
**Pull request by likewei92**
Including an arc-exclude input that exclude the element and all its children

### CODE REVIEW

File: src/focus.service.ts [hunk 1]
```diff
@@ -554,6 +554,13 @@ export class FocusService {
       return false;
     }
 
+    for (let parent = el; parent; parent = parent.parentElement) {
+      const record2 = this.registry.find(parent);
```
Comment (connor4312) [inline src/focus.service.ts]: You can name this `record` instead of `record2`. const/let declarations are block-scoped.

Comment (SimonSchick) [inline src/focus.service.ts]: IIRC our linter settings forbid shadowing (which is a good thing), should choose a different name tho.

Comment (connor4312) [inline src/focus.service.ts]: ah, yea, misread over github's folding line. A better name would be better though

Comment (likewei92) [inline src/focus.service.ts]: renamed

**Pull request merged**